exonum/src/node/consensus.rs

```diff
@@ -857,6 +857,14 @@ impl NodeHandler {
         if let Some(peer) = self.state.retry(data, peer) {
             self.add_request_timeout(data.clone(), Some(peer));
 
+            if !self.is_enabled {
+                trace!(
+                    "Do not send a request {:?} because the node is paused.",
```

Comment:
Nit: "Not sending a request..." makes more sense from the grammar perspective, I believe.